go/pkg/client/client.go

```diff
@@ -234,6 +237,16 @@ func (s CompressedBytestreamThreshold) Apply(c *Client) {
 	c.CompressedBytestreamThreshold = s
 }
 
+// An UploadCompressionClassifier determines what kind of compression to use for a blob on upload.
```

Comment:
This comment line is out of date. Also, I wonder if `UploadCompressionPredicate` is a more suitable name now (which would also leave the "classifier" suffix available for a future option).

Reply:
Yup, makes sense. Done.